index.html

```diff
@@ -10,6 +10,24 @@
       
     </title>
     <meta name="viewport" content="width=device-width">
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Assetnote Wordlists">
+    <meta name="description" content="Wordlists that are up to date and effective against the most popular technologies on the internet.">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://wordlists.assetnote.io/">
+    <meta property="og:title" content="Assetnote Wordlists">
+    <meta property="og:description" content="Wordlists that are up to date and effective against the most popular technologies on the internet.">
+    <meta property="og:image" content="https://securib.ee/images/assetnote.png">
```

Comment:
Can we please change this to: `https://wordlists.assetnote.io/assets/assetnote.png`?